js/controller.js

```diff
@@ -274,6 +271,14 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.mainVideoElement.css(InlineStyle.pauseScreenStyle.videoUnblur);
         this.renderSkin();
       }
+      if (source == OO.VIDEO.ADS) {
```

Comment:
Yes, every time ad or main video starts playing, we get the VC_PLAYING event. I tested skipping the ads also.